### ISSUE TITLE
Home: Could not load PIO Account information

### ISSUE THREAD

**Issue by RayEbb** [platformio/platformio-core#1464]
Server error. I did a password reset, but i can't login with the temp Password.

**Comment by ivankravets** [comment]
Please open PIO IDE Terminal and run `pio upgrade`. Restart IDE. Does it work now?

**Comment by RayEbb** [comment]
Thank you very much, IvanKravets!! I tried to start the PIO terminal, but the only terminal i saw, was the CMD.  That's why i uninstalled de PIO IDE, and reinstalled it again. I think i have to setup Visual Studio Code first, and the Workbench also. But it's very tricky for me, to be honest. So many settings, lol... And now, i found the little PIO icons on the bottom of the screen, with the Terminal at the end of the line.... 

And now, IT WORKS!!!! 
Stupid me, i know.. :D But, I've learned something, thanks to you!!!
